### ISSUE TITLE
Offering -> more about the organisation

### ISSUE THREAD

**Issue by Leptopoda** [akwSquad/aufklaerwerk-react#25]
When hitting "more about the organisation" the entire page reloads.
This triggers the animations to be shown again and reloads the google maps integration.

**Comment by Leptopoda** [comment]
But can't this be shown without the animation replaying?

**Comment by felixhasse** [comment]
Yes, that's definitely possible, but idk if we want this since it's a different page. I will ask the Aufklärwerk Team.
